build.gradle

```diff
@@ -227,6 +235,24 @@ task setupEsCluster() {
     }
 }
 
+/**
+ * integTestRunner is a task provided by the ES test framework, which allows us to spin up clients
+ * and test API calls against a local or remote Elasticsearch cluster.
+ *
+ * The simplest way to run this task in a way that "just works" is to invoke
+ * ./gradlew integTest -Dtests.enableIT=true -Dtests.useDockerCluster=true
+ * which will spin up a local 2 node ES cluster on your machine, then execute the test suite against it
+ *
+ * A full list of options is provided below. Check our gradle.properties file for the defaults for
+ * each of these properties.
+ *
+ * -Dtests.rest.cluster the Elasticsearch REST endpoint that test clients should hit
+ * -Dtests.cluster the Elasticsearch transport endpoint that test clients should hit
+ * -Dtests.enableIT a flag to enable integration testing, by default this is false
+ * -Dtests.useDockerCluster if true, spin up a local 2 node cluster before executing tests
+ *        NOTE: if you specify this, don't specify -Dtests.rest.cluster or -Dtests.cluster
+ * -Dtests.pa.port the port number of the PerformanceAnalyzer REST endpoint
+ */
```

Comment:
We haven't added your latest option `tests.https`, `tests.user` and `tests.password`  :)

Reply:
Really good catch, I tried to make this recent commit more comprehensive